src/main/java/io/confluent/idesidecar/restapi/kafkarest/controllers/ClusterManager.java

```diff
@@ -0,0 +1,15 @@
+package io.confluent.idesidecar.restapi.kafkarest.controllers;
```

Comment:
Is this package meant to hold controller classes per the MVC pattern?

Reply:
No, I don't think it's meant to represent MVC. Perhaps just calling the package `services` would be more appropriate? WDYT?

Reply:
Did you consider not having any subpackages under `kafkarest`? As mentioned my other comment, the filter could very easily go to `...restapi.filters`, and the `AdminClientInstantiationException` to `...restapi.exceptions`. That leaves just the `impl` and `controllers` packages for a total of eight classes.

Reply:
I think that's fine. I'll make the eight classes be top-level under `kafkarest` and move the other two out.